components/common/src/main/java/com/hotels/styx/api/metrics/codahale/CodaHaleMetricRegistry.java

```diff
@@ -65,50 +128,76 @@ public MetricRegistry scope(String name) {
 
     @Override
     public <T extends Metric> T register(String name, T metric) throws IllegalArgumentException {
-        return metricRegistry.register(name, metric);
+        Tags tags = Tags.of("metricSource", "dropwizard");
```

Comment:
Unused here, and I think you handle this in addAttributes anyway.

Reply:
Done